tools/sesman/sesmanworker/engine.py

```diff
@@ -614,10 +618,31 @@ def notify_find_process_rdp(self, regex, deny, app_name, app_cmd_line,
             Logger().info("Engine NotifyFindProcessInRDPFlow failed: "
                           "(((%s)))" % (traceback.format_exc()))
 
+    def _get_target_filter_mode(self, device_filter):
+        if not device_filter:
```

Comment:
if group or protocol filter is not empty, it might be TargetFilterMode.NORMAL.

Reply:
So, even in "device_filter" parameter we have "?account=my_account" which is a SelectorFilterMode.ADVANCED, it will be set to SelectorFilterMode.NORMAL?

It means the advanced selector will not running if group or protocol is set. Is it ok?

I thought to put directly the check of group/protocol and the set of the mode outside of get_selector_filter_mode() and directly in get_targets_list() for keep same code of the targetaccuratefilter.py module with SSH Proxy